run-build.sh

```diff
@@ -164,7 +164,7 @@ if [ $EXIT_CODE != 0 ]; then
 fi
 
 # Install a project.json based CLI for use by tests
-(set -x ; "$REPOROOT/scripts/obtain/dotnet-install.sh" --channel "master" --install-dir "$DOTNET_INSTALL_DIR_PJ" --architecture "$ARCHITECTURE" --version "1.0.0-preview2-1-003177")
+(set -x ; "$REPOROOT/scripts/obtain/dotnet-install.sh" --channel "master" --install-dir "$DOTNET_INSTALL_DIR_PJ" --architecture "$ARCHITECTURE" --version "2.1.0-preview1-007012")
```

Comment:
The fact that things are green with this change applied suggests to me that we no longer actually use the project.json stage 0 for anything. 2.1.x doesn't support project.json. The whole point here was to get an old pre-1.0 version in order to do some things with project.json.

I think we should remove this altogether along with other references to Stage0PJ, PjDotnetPath, etc.

Reply:
This is being removed in #7522 by @dsplaisted, so it will be fixed when that is merged.